boot/storage.txt

```diff
@@ -0,0 +1,424 @@
+    +-------------------------------+------------------------------------------+
+    |                               |                                          |
+    |    GPT + BIOS (with swap)     |   /dev/sda1 (BIOS BOOT, EXT4,   128MB)   |
+    |                               |   /dev/sda2 (UEFI ESP,  FAT32,  33M)     |
+    |                               |   /dev/sda3 (swap,      swap,   >512MB)  |
+    |                               |   /dev/sda4 (ROOTFS,    EXT4,   *)       |
+    |                               |                                          |
+    +-------------------------------+------------------------------------------+
+    
+    * In most cases, the user will desire the root partition to span the remain 
+      space on the disk device.
+    * Building big packages like rust or firefox from source require a large 
+      amount of memory. Make sure that your physical RAM and swap add up to 
+      ~10GB or more of size. Or consider the usage of a dynamical growing swap 
+      file.
+    * A separate /home partition may be preferred, especially if you plan on 
+      re-installing KISS Linux in the future and would like to preserve any 
+      custom configuration files, downloads, etc.
+
+      
+    [2.2] Swap Partition or Swap File
+    ____________________________________________________________________________
+    
+    (placeholder text)
+    
+    
+    [2.3] EFI/UEFI or BIOS
+    ____________________________________________________________________________
+
+    Depending on your chosen boot process type UEFI or BIOS, you may need to 
+    create additional partitions. Typically, these additional partitions are
+    created before specifying the root partition.  
+    
+    - For UEFI, a ESP partition must be created and FAT formatted (preferably 
+      FAT32) [1].
+    - BIOS boot partition is needed when a GPT partition layout is used with 
+      GRUB2 in BIOS/Legacy mode. It is not required when booting in EFI/UEFI 
+      mode.
+    
+    
+[3.0] Formatting
+________________________________________________________________________________
+
+Now that the partitions are created, it is time to place a filesystem on them. 
+In the next section the various file systems that Linux supports are described.
+
+
+    [3.1] Filesystems
+    ____________________________________________________________________________
+
+    (place holder text)
+
+
+    [3.2] Formatting a Disk
+    ____________________________________________________________________________
+    
+    In order for a disk to be used as an installation medium, the storage space 
+    must be formatted to the desired file system type.  The following are 
+    example commands that are useful for creating those partition types. Note 
+    that not all commands will be available on your distribution Live CD.  For 
+    this reason, I would recommend consulting the individuals distribution help 
+    pages.
```

Comment:
```
For this reason, I would recommend consulting the individuals distribution help pages.
```

`individuals distribution` -> `live CD distribution's` (this sounds a little wrong too)

What help pages? Maybe documentation is a better word?